code/modules/mob/living/carbon/superior_animal/giant_spider/types/nurse.dm

```diff
@@ -97,7 +97,7 @@
 	icon_living = "recluse"
 	maxHealth = 15
 	health = 15
-	poison_per_bite = 1 //1u is all it takes to sleep you, your asleep also dosnt prevet it form attacking you more then once meaning this quit quickly stacks without someones help
+	poison_per_bite = 0.5 //Lets not have a classic ss13 traitor chem be used as an instant down when people are lagging. 2 hits for your unfun spider
```

Comment:
```suggestion
	poison_per_bite = 1
```

Seeing as how we have the armor reduces injection code, both reducing the dose *and* adding the code that makes doses lower than 1 not take effect, it isn't necessary to do both. If you go into maints without armor, that's an unforced error.